15-7-1_and_15-c-1/compile_tables.R

```diff
@@ -0,0 +1,49 @@
+# This file is directly called by update_indicator_main.R.
+# It has to be called compile_tables.R
+# It is the control script that runs all the others.
+
+library('openxlsx')
+library('stringr')
+library('janitor')
+library('tidyr')
+library('dplyr')
+
+library(SDGupdater)
+
+source("15-7-1_15-c-1_example_config.R") # pulls in all the configurations. Change to "config.R" for real update
```

Comment:
This can be commented out when the if statement above is added. It'll source the correct config automatically.

Reply:
Done